runtime/cudaq/qis/qubit_qis.h

```diff
@@ -36,6 +36,7 @@ namespace qubit_op {
 ConcreteQubitOp(h) ConcreteQubitOp(x) ConcreteQubitOp(y) ConcreteQubitOp(z)
     ConcreteQubitOp(s) ConcreteQubitOp(t) ConcreteQubitOp(rx)
         ConcreteQubitOp(ry) ConcreteQubitOp(rz) ConcreteQubitOp(r1)
+            ConcreteQubitOp(u3)
```

Comment:
Should we be adding `u2` here while we're at it?

Reply:
No, I don't plan to. U2 can be implemented with U3 if required.